workflows/prognostic_c48_run/tests/test_monitor.py

```diff
@@ -0,0 +1,15 @@
+import xarray
+from runtime.monitor import Monitor
+from runtime.names import DELP
+
+
+def test_Monitor_monitor():
+    ds = xarray.Dataset({"x": ([], 0.0), "y": (["z"], [1, 2]), DELP: (["z"], [1, 1])})
+    tend = "tendency_of_x_due_to_blah"
+    storage = "storage_of_y_path_due_to_blah"
+
+    monitor = Monitor([tend, storage], ds, timestep=900)
```

Comment:
Is this expected to pass, given that "blah" does not exist in the dataset/state? I would think this should raise an exception. If this is expected, could you add it as a special test and make the default case use an existing variable? This would help avoid confusing someone reading the tests.

Reply:
Yes. The name "blah" is specified below.

Reply:
Hopefully f86baf3 clarifies how this works. Let me know if it is still confusing.

Reply:
I'm still a bit confused. How can Monitor monitor a variable "blah" that is not defined?

Reply:
Ah my confusion was in thinking "blah" is a variable when it's not. This is the same issue as the `__init__` confusion. Resolved.